src/graphql/authentication.ts

```diff
@@ -26,3 +26,7 @@ export const generateToken = async (id: string, name: string, rememberMe: boolea
 
   return jwt.sign(payload, secret, { expiresIn: expirationTime });
 };
+
+export const tokenValidation = (token: string) => {
```

Comment:
```suggestion
export const isTokenValid = (token: string) => {
```

Geralmente é bom ter o prefixo is em funções que retornam booleans. Enunciar o nome da função como se fosse uma pergunta de sim ou não.